extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeCommentsExtractor.java

```diff
@@ -105,6 +104,16 @@ private String findInitialCommentsToken(final JsonObject nextResponse)
         return token;
     }
 
+    @Nonnull
+    private JsonArray getJsonContents(final JsonObject nextResponse) {
+        try {
+            return JsonUtils.getArray(nextResponse,
+                    "contents.twoColumnWatchNextResults.results.results.contents");
+        } catch (final ParsingException pe) {
+            return new JsonArray(Collections.emptyList());
```

Comment:
Why do you want to use the constructor with a `Collection` where you pass an empty list? Can't you just use an empty `JsonArray` instead?

Reply:
It's a coding convention I've become accustomed to. The line of thinking is that the function returns the content of the JSON, and should be read-only.

Reply:
Ok :)